docs/en/quantize.md

```diff
@@ -1,11 +1,77 @@
-[English](../en/quantize.md) | 简体中文
+[简体中文](../cn/quantize.md) ｜English 
 
-# 量化加速
+# Quantization Acceleration
 
-简要介绍量化加速的原理。
+Quantization is a popular method in model compression to obtain models with smaller sizes and faster inference speeds.
+Based on PaddleSlim, FastDeploy integrates a one-click model quantization tool. FastDeploy supports the deployment of quantized models for faster inference.
 
-目前量化支持在哪些硬件及后端的使用
+## FastDeploy supports quantitative model deployment for multiple engines and hardware
 
-## 量化示例
+Currently, multiple inference backends in FastDeploy can deploy quantitative models on different hardware. 
 
-这里一个表格，展示目前支持的量化列表(跳转到相应的example下去)，精度、性能
+| Hardware / Inference Backend | ONNX Runtime | Paddle Inference | TensorRT |
+|:---------------------------- |:------------ |:---------------- |:-------- |
+| CPU                          | Support      | Support          |          |
+| GPU                          |              |                  | Support  |
+
+## Model Quantization
+
+### Quantization Methods
+
+Based on PaddleSlim, FastDeploy currently offers knowledge distillation training and post-training quantization. Knowledge distillation training quantizes models by model training, and post-training quantization can complete the model quantization without training. FastDeploy can deploy quantization models from both approaches.
+
+The comparison between both methods is as follows.
+
+| Method                          | Time                                            | Accuracy                              | Size     | Inference Speed |
+| ------------------------------- | ----------------------------------------------- | ------------------------------------- | -------- | --------------- |
+| Post-training quantization      | No training needed, fast deployment             | Slightly lower than distillation      | The same | The same        |
+| Knowledge distillation training | Training needed, and takes slightly longer time | Slight lower than un-quantized models | The same | The same        |
+
+### Use FastDeploy One-Click Quantization Tool
+
+Fastdeploy, based on PaddleSlim, provides users with a one-click model quantification tool. Please refer to the following document for model quantification.
+
+- [FastDeploy One-Click Quantization Tool](../../tools/quantization/)
+  Once developers obtain a quantitative model, they can use FastDeploy to deploy the quantized model.
```

Comment:
quantitative --> quantized

Reply:
Once developers obtain a quantized model, they can use FastDeploy to deploy them to gain speedup.